Dockerfile

```diff
@@ -1,10 +1,12 @@
 FROM golang:1.15-alpine
+RUN apk --no-cache add curl
 
 WORKDIR /go/src/app
 COPY . .
 
+RUN curl --silent "https://api.github.com/repos/datreeio/datree/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/' > cli-version
 RUN go get -d -v ./...
```

Comment:
Melech!
I think it will be a better idea saving the version as a variable rather than as a file.
something like `cli_version=$(curl --silent "https://api.github.com/repos/datreeio/datree/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')`

Reply:
not possible. docker evaluates each layer seperately and saves image state when moving from one command to another. there's no memory sharing